src/System.Management.Automation/engine/hostifaces/PowerShellProcessInstance.cs

```diff
@@ -91,8 +99,20 @@ public PowerShellProcessInstance(Version powerShellVersion, PSCredential credent
         }
 
         /// <summary>
+        /// Initializes a new instance of the <see cref="PowerShellProcessInstance"/> class. Initializes the underlying dotnet process class. 
+        /// </summary>
+        /// <param name="powerShellVersion"></param>
+        /// <param name="credential"></param>
+        /// <param name="initializationScript"></param>
+        /// <param name="useWow64"></param>
+        public PowerShellProcessInstance(Version powerShellVersion, PSCredential credential, ScriptBlock initializationScript, bool useWow64) : this(powerShellVersion, credential, initializationScript, useWow64, null)
```

Comment:
```suggestion
        public PowerShellProcessInstance(Version powerShellVersion, PSCredential credential, ScriptBlock initializationScript, bool useWow64) : this(powerShellVersion, credential, initializationScript, useWow64, workingDirectory: null)
```

Reply:
Please use named parameters

Reply:
fixed in 5a5efd4f1d00333cea56bdc09f9c4891654afa7f.